srcalc/src/test/java/gov/va/med/srcalc/domain/model/RiskModelTest.java

```diff
@@ -119,6 +121,48 @@ public final void testCalculateDuplicateValues() throws Exception
     }
     
     @Test
+    public final void testEquals()
+    {
+        // This isn't helpfull when its test cases include impossible cases. One example sets a string ("red") as 
+        // one of the terms even though our getTerms() method returns ModelTerms.
+//        EqualsVerifier.forClass(RiskModel.class).
+//             suppress(Warning.NULL_FIELDS,Warning.NONFINAL_FIELDS).verify();
```

Comment:
kept the commented out code but wrote more explanation as to why it wasn't used.